src/main/java/net/sf/javaanpr/intelligence/RecognizedPlate.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2013 JavaANPR contributors
+ * Copyright 2016 JavaANPR contributors
```

Comment:
I'm also not a lawyer, but I remember that when I worked for one of the IT corporations and we decided to move one of the projects to GPL we had to keep this up to date and track all authors in the file header comment (who modified given file, when and why).

I'm really not sure if this is required.